tests/src/e2e.spec.ts

```diff
@@ -54,34 +98,92 @@ test.skip('Check welcome page is displayed and that we are redirected to the mai
          * The main window is created hidden, and is shown only when it is ready.
          * See {@link ../packages/main/src/mainWindow.ts} function
          */
+
+        mainWindow.webContents.closeDevTools();
         if (mainWindow.isVisible()) {
           resolve(getState());
         } else mainWindow.once('ready-to-show', () => resolve(getState()));
       });
-    },
-  );
+    });
+    expect(windowState.isCrashed, 'The app has crashed').toBeFalsy();
+    expect(windowState.isVisible, 'The main window was not visible').toBeTruthy();
 
-  await page.screenshot({ path: 'tests/output/screenshots/screenshot-welcome-page-init.png', fullPage: true });
+    await page.screenshot({ path: 'tests/output/screenshots/screenshot-welcome-page-init.png', fullPage: true });
 
-  // wait for the initial screen to be loaded
-  const goToPodmanDesktopButton = page.locator('button:text("Go to Podman Desktop")');
-  // wait for visibility
-  await goToPodmanDesktopButton.waitFor({ state: 'visible' });
+    const welcomeMessage = page.locator('text=/Welcome to Podman Desktop.*/');
```

Comment:
why have you changed the test that was checking the 'Go to Podman Desktop' ?

Reply:
Was just moved a bit below. So the testing workflow could be extended.